.travis.yml

```diff
@@ -40,6 +41,7 @@ matrix:
 
 before_install:
   - make ci-docker-load
+  - make travis-install-protobuf
```

Comment:
no no this is the convention, see `make travis-docker-push`, this is specific to travis